config/initializers/admins.rb

```diff
@@ -0,0 +1,4 @@
+module Gemcutter
+  mattr_accessor :admins
+  self.admins = Application.config_for("admins")["emails"]
+end
```

Comment:
If admins aren't setup, the app shouldn't boot. It's a configuration error from the get go. Much the same as something missing from `database.yml`.